docs/configuration/single-process-config.md

```diff
@@ -9,7 +9,13 @@ Configuration for running Cortex in single-process mode.
 This should not be used in production.  It is only for getting started
 and development.
 
+[embedmd]:# (./single-process-config.yaml)
 ```yaml
+
+# Configuration for running Cortex in single-process mode.
+# This should not be used in production.  It is only for getting started
```

Comment:
By “this” — do you mean this example configuration, or “single process” mode in general? Should be clarified.

I think we want to encourage people to use single-process mode in production.

Reply:
It means the config. Updated.